src/types/requests.ts

```diff
@@ -1,43 +1,43 @@
 import { TokenAmount } from './tokens';
-import { FeeEntry } from '../api';
+import { FeeEntry, MintRequest } from '../api';
 
-// These custom interfaces are used because API.SignableToken is not yet a union type due to OAS 2.0 spec not supporting `OneOf`
+// These custom request interfaces are used because API.SignableToken is not yet a union type due to OAS 2.0 spec not supporting `OneOf`
 // As well as the fact that SignableToken has an extra `data` object which we would like to flatten
 // SignableToken is replaced with AnyToken
 
+/**
+ * Parameter required to create an Order
+ */
 export interface UnsignedOrderRequest {
+  /**
+   * The amount of tokens that will be bought for this order
+   */
   buy: TokenAmount;
+  /**
+   * The amount of tokens that will be sold for this order
+   */
   sell: TokenAmount;
   /**
    * ExpirationTimestamp in Unix time. Note: will be rounded down to the nearest hour
-   * @type {number}
-   * @memberof UnsignedOrderRequest
    */
   expiration_timestamp?: number;
   /**
    * Inclusion of either maker or taker fees
-   * @type {Array<FeeEntry>}
-   * @memberof UnsignedOrderRequest
    */
   fees?: Array<FeeEntry>;
 }
 
+/**
+ * Parameter required to create a Transfer
+ */
 export type UnsignedTransferRequest = TokenAmount & {
   /**
    * Ethereum address of the receiving user
-   * @type {string}
-   * @memberof UnsignedTransferRequest
    */
   receiver: string;
 };
 
-export interface NftTransferDetails {
-  /**
-   * Ethereum address of the receiving user
-   * @type {string}
-   * @memberof NftTransferDetails
-   */
-  receiver: string;
-  tokenId: string;
-  tokenAddress: string;
-}
+/**
+ * Parameter required to mint tokens
```

Comment:
```suggestion
 * Parameters required to mint tokens
```

Reply:
Hmm, I thought about this. The interface is technically a `parameter`.. that contains `parameters`. I think non-plural makes sense?